src/Compilers/CSharp/Portable/BoundTree/DecisionTreeBuilder.cs

```diff
@@ -165,15 +159,16 @@ private DecisionTree AddByValue(DecisionTree.ByType byType, BoundConstantPattern
                 {
                     if (byType.Default.MatchIsComplete)
                     {
+                        // This code may be unreachable due to https://github.com/dotnet/roslyn/issues/16878
```

Comment:
Please at least add an explicit note about this code path in the issue so that we definitely address it, even if we decide to do nothing about sub-optimal code generation. Ideally, I would prefer us to track code coverage in a separate issue, not mixing it with issues about sub-optimal behaviors.